src/main/webapp/WEB-INF/tags/instructor/course/studentsTable.tag

```diff
@@ -0,0 +1,78 @@
+<%@ tag description="instructorCourseDetails - Course Information Board" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="courseDetails" type="teammates.common.datatransfer.CourseDetailsBundle" required="true" %>
+<%@ attribute name="studentsTable" type="teammates.ui.template.CourseDetailsStudentsTable" required="true" %>
+
+<table class="table table-bordered table-striped">
+    <c:set var="hasSection">
+        ${courseDetails.stats.sectionsTotal > 0}
+    </c:set>
```

Comment:
May be more clean to use a single line for this:
`<c:set var="hasSection" value="${courseDetails.stats.sectionsTotal > 0}" />